packages/status-js/src/client/chat.ts

```diff
@@ -203,26 +202,25 @@ export class Chat {
       callback: (wakuMessages: WakuMessage[]) => {
         let index = wakuMessages.length
 
-        this.#fetchingMessages = true
```

Comment:
I removed the flag in favour of `source` out of worry that it could be blocking should fetch ever not be immediately `await`ed and take longer (e.g. "Last 7 days").